include/micm/solver/cuda_linear_solver.hpp

```diff
@@ -86,7 +86,7 @@ namespace micm
         const SparseMatrixPolicy<T>& L,
         const SparseMatrixPolicy<T>& U) const
     {
-      LinearSolver<T, SparseMatrixPolicy, LuDecompositionPolicy>::Solve<MatrixPolicy>(b, x, L, U);
+      LinearSolver<T, SparseMatrixPolicy, LuDecompositionPolicy>:: template Solve<MatrixPolicy>(b, x, L, U);
```

Comment:
Interesting. Why we need the `template` keyword here but not in other CUDA c++ files?

Reply:
I need to dig more into this myself but this was the error (https://github.com/NCAR/micm/actions/runs/9001800054/job/24728778386#step:4:345) and an explanation behind this was https://stackoverflow.com/a/34732843 which I need to dig more into myself.